src/workerd/api/crypto/scrypt.c++

```diff
@@ -3,13 +3,19 @@
 //     https://opensource.org/licenses/Apache-2.0
 
 #include "impl.h"
+#include "kdf.h"
 
-#include <workerd/api/crypto/kdf.h>
-
-#include <openssl/evp.h>
+#include <ncrypto.h>
 
 namespace workerd::api {
 
+namespace {
+template <typename T = const kj::byte>
```

Comment:
I see that this function is defined in multiple files. I recommend moving to a common denominator

Reply:
It will be.